### PR TITLE
Appx.A Testing Tools Resource - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR covers issue #90 

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting errors
- Fix dead links

### CODE REVIEW

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md [hunk 1]
```diff
@@ -1,274 +1,215 @@
...
 
 #### Testing for JavaScript Security, DOM XSS
 
--   BlueClosure BC Detect - <http://www.blueclosure.com>
+- **[BlueClosure BC Detect](http://www.blueclosure.com)**
 
 #### Testing AJAX
 
--   **[OWASP Sprajax Project](:Category:OWASP_Sprajax_Project "wikilink")**
+- **[OWASP Sprajax Project](https://www.owasp.org/index.php/:Category:OWASP_Sprajax_Project)**
```
Comment (ThunderSon) [inline document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: If we are to keep it, remove the `:` before `Category` please

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md [hunk 2]
```diff
@@ -1,274 +1,215 @@
...
--   Sqlninja: a SQL Server Injection & Takeover Tool - <http://sqlninja.sourceforge.net>
--   Bernardo Damele A. G.: sqlmap, automatic SQL injection tool - <http://sqlmap.org/>
--   Absinthe 1.1 (formerly SQLSqueal) - <http://sourceforge.net/projects/absinthe/>
--   SQLInjector - Uses inference techniques to extract data and determine the backend database server. <http://www.databasesecurity.com/sql-injector.htm>
--   Bsqlbf-v2: A perl script allows extraction of data from Blind SQL Injections - <http://code.google.com/p/bsqlbf-v2/>
--   Pangolin: An automatic SQL injection penetration testing tool - <http://www.darknet.org.uk/2009/05/pangolin-automatic-sql-injection-tool/>
--   Antonio Parata: Dump Files by sql inference on Mysql - SqlDumper - <http://www.ruizata.com/>
--   Multiple DBMS Sql Injection tool - SQL Power Injector - <http://www.sqlpowerinjector.com/>
--   MySql Blind Injection Bruteforcing, Reversing.org - sqlbftools - <http://packetstormsecurity.org/files/43795/sqlbftools-1.2.tar.gz.html>
+- **[OWASP SQLiX](https://www.owasp.org/index.php/:Category:OWASP_SQLiX_Project)**
```
Comment (ThunderSon) [inline document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Why are they in bold formatting? And please remove the `:` before the `Category`

Comment (rejahrehim) [reply document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Done

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md [hunk 3]
```diff
@@ -1,274 +1,215 @@
...
--   '''Session Manager"" - <https://chrome.google.com/webstore/detail/bbcnbpafconjjigibnhbfmmgdbbkcjfi>
-    -   With Session Manager you can quickly save your current browser state and reload it whenever necessary. You can manage multiple sessions, rename or remove them from the session library. Each session remembers the state of the browser at its creation time, i.e the opened tabs and windows.
--   **Subgraph Vega** - <http://www.subgraph.com/products.html>
-    -   Vega is a free and open source scanner and testing platform to test the security of web applications. Vega can help you find and validate SQL Injection, Cross-Site Scripting (XSS), inadvertently disclosed sensitive information, and other vulnerabilities. It is written in Java, GUI based, and runs on Linux, OS X, and Windows.
+# Open Source Black Box Testing tools
```
Comment (kingthorin) [inline document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Title cap "Tools" please

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md [hunk 4]
```diff
@@ -1,274 +1,215 @@
...
+  - Edit This Cookie is a cookie manager. You can add, delete, edit, search, protect and block cookies
+- [Cookie swap](https://chrome.google.com/webstore/detail/dffhipnliikkblkhpjapbecpmoilcama?hl=en-US)
+  - Swap My Cookies is a session manager, it manages cookies, letting you login on any website with several different accounts.
+- [Session Manager](https://chrome.google.com/webstore/detail/session-manager/mghenlmbmjcpehccoangkdpagbcbkdpc)
+  - With Session Manager you can quickly save your current browser state and reload it whenever necessary. You can manage multiple sessions, rename or remove them from the session library. Each session remembers the state of the browser at its creation time, i.e the opened tabs and windows.
+- [Subgraph Vega](http://www.subgraph.com/products.html)
+  - Vega is a free and open source scanner and testing platform to test the security of web applications. Vega can help you find and validate SQL Injection, Cross-Site Scripting (XSS), inadvertently disclosed sensitive information, and other vulnerabilities. It is written in Java, GUI based, and runs on Linux, OS X, and Windows.
 
 ### Testing for specific vulnerabilities
```
Comment (kingthorin) [inline document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Title caps please (through out)

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md [hunk 5]
```diff
@@ -1,274 +1,215 @@
...
--   bugtraq <http://bugtraq-team.com/>
-
-Source Code Analyzers
----------------------
+- [Slowloris](https://github.com/gkbrk/slowloris)
+- [slowhttptest](https://github.com/shekyan/slowhttptest)
+
+## Commercial Black Box Testing tools
+
+- [NGS Typhon](https://www.nccgroup.trust/uk/our-services/security-consulting/information-security-software/?utm_source=marketing&utm_medium=rd0517)
```
Comment (kingthorin) [inline document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Please remove the `utm` params.

Comment (rejahrehim) [reply document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Without `utm` params it redirects to the home page removed one

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md (hunk 4 above)
Comment (rejahrehim) [reply document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Done

File: document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md [hunk 6]
```diff
@@ -121,9 +121,9 @@
 - [Slowloris](https://github.com/gkbrk/slowloris)
 - [slowhttptest](https://github.com/shekyan/slowhttptest)
 
-## Commercial Black Box Testing tools
+## Commercial Black Box Testing Tools
 
-- [NGS Typhon](https://www.nccgroup.trust/uk/our-services/security-consulting/information-security-software/?utm_source=marketing&utm_medium=rd0517)
+- [NGS Typhon](https://www.nccgroup.trust/uk/our-services/security-consulting/information-security-software/?utm_medium=rd0517)
```
Comment (kingthorin) [inline document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Both?

Comment (rejahrehim) [reply document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: If we remove utm_medium=rd0517 it will redirect to home

Comment (kingthorin) [reply document/Appx.A_Testing_Tools_Resource/Appx.A_Testing_Tools.md]: Hmmm alright.

**Comment by rejahrehim** [comment]
@ThunderSon Check this PR too.

**Comment by ThunderSon** [comment]
@rejahrehim Haha the button next to my name: Re-request review. It will appear in a different manner on GitHub, and it's faster for you to do 😋

**Pull request merged**